Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1569,6 +1580,73 @@ - (IBAction)showImageChoices:(id)sender {
     [self beginTaskWithIdentifier:ImageChoicesTaskIdentifier];
 }
 
+# pragma mark - Image Capture
+- (id<ORKTask>)makeImageCaptureTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"begin"];
+        step.title = @"Hands";
+        step.image = [UIImage imageNamed:@"hands_solid"];
```

Comment:
If you mean do something like this:
```
        step.image = [[UIImage imageNamed:@"hands_solid" inBundle:nil compatibleWithTraitCollection:nil] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
```
If so, this will unfortunately not make the hands show up as blue in the instruction step since the `shouldTintImages` in the `ORKInstructionStep` will be false by default, and so the `ORKTintedImageView` will recreate the image with `UIImageRenderingModeAutomatic` before showing it.  This is the issue I was referring to earlier regarding hacking `ORKInstructionStep` so I could get at this property.